packages/swimlane/changelog.yml

```
@@ -0,0 +1,6 @@
+# newer versions go on top
+- version: "0.0.1"
```

```suggestion
- version: "0.1.0"
```
Change this version inside `manifest.yml` as well.